public/views/join.html

```diff
@@ -124,8 +124,8 @@
 
             <div ng-show="join.seedSourceId=='set'">
               <label for="ext-master">
-                <span translate>Wallet Seed</span>
-                <small translate>Enter the seed words (BIP39)</small>
+                <span translate>Wallet Recovery Phrase</span>
+                <small translate>Enter the recovery words (BIP39)</small>
```

Comment:
I know this inconsistency existed before but maybe we need to stick with "recovery phrase" and remove all references to "recovery words"?